lib/tasks/tasks.rake

```diff
@@ -123,6 +119,18 @@ namespace :synthea do
     end
   end
 
+  def clear_output
+    if Synthea::Config.sequential.clean_output_each_run
+      puts 'Clearing output folders...'
+      %w(html fhir CCDA text csv).each do |type|
+        out_dir = Synthea::Output::Exporter.get_output_folder(type)
+        FileUtils.rm_r out_dir if File.exist? out_dir
+        FileUtils.mkdir_p out_dir
+      end
+      Synthea::Output::CsvRecord.open_csv_files
```

Comment:
this call should be outside the `if Synthea::Config.sequential.clean_output_each_run` and should instead be conditional based on `Synthea::Config.exporter.csv.export`

also the effect of opening files isn't implied by the name "clear_output". Not a huge deal but I'd rather the call be directly inside the generate and sequential tasks, so it's clear what's happening and it's symmetric with the call to `close_csv_files` (which should also be conditional based on `Synthea::Config.exporter.csv.export` if the open call is)